### DISCUSSION TITLE
values entered for Source Type on Age & History do not appear to be saved

### DISCUSSION THREAD

**Discussion by Toase-of-London** [colouring-cities/colouring-britain#31] [Bug Reports]
HI. When using the Age & History set of values, when I select a value for Source Type from the drop down and click on 'Save' the message "Database Error" is returned. Clicking away to another building and then coming back to the original one show "Source Tye" is blank.
Observed since I''ve been using the Source Type field (about two weeks) and most recently happened today (29-Jul-23).

**Comment by Toase-of-London** [comment]
I have just repeated the process outlined above and it looks like "historical source type" is not saved either. I have a suspicion this has worked before, since I can check buildings I've edited before and values I have entered in the"historical source type" field are visible.

**Comment by mdsimpson42** [comment]
Hi. Yes, any fields that you changed at the same time as the database error occurred were likely to be lost. I believe this is now fixed. Do let us know if you spot any other bugs (we do our best but we can't always test everything!).

**Reply by Toase-of-London** [reply]
Agreed Mike, it is very rarely possible to test everything in anything other than the simplest applications. In my professional life I am a software test manager and always have to make judgement calls about where best to allocate testing resource. So I totally understand how issues can slip through. Also, user always do weird things that are hard to anticipate so it's interesting for me to be on the user side this time. I promise to raise issues with as much information as possible so they are easier to solve.
Kind regards
Chris
